Various Fixes

thetwopct opened the pull request:
- Updating pagination, reduced numbers shown on mobile, fixes #328
- Reverting archive text size on mobile, applying small excerpt on posts other than top feature post on tablet and up, closes #321 
- Removing Pre-publish checklists, removing YouTube Lite (wrong plugin)
- Updating LF MU to remove pre-publish checklists
- Adding additional image sizes to hero, ref #309
- Adding aria label to People modal (small accessibility improvement)

More to come

cjyabraham commented:
I think it looks a bit weird having just one page number in the mobile view:
<img width="415" alt="Screen Shot 2020-09-18 at 3 12 13 PM" src="https://user-images.githubusercontent.com/463160/93567554-6400cd80-f9c1-11ea-86a6-d6d8237d6c0c.png">

When you go from desktop to mobile you wonder where the other pages have gone.

thetwopct commented:
Added:
- Event has passed message to single events. [Example](https://pr-330-cncfci.pantheonsite.io/events/openjs-world-2020/).
- Improving search in the header dropdown. [Example](https://pr-330-cncfci.pantheonsite.io/?s=chris).
- Changed the Back to Top button CSS slightly - please see if this fixes the problem on your device.

thetwopct commented:
Added: 

- Refactor of external link icons (no line wrap of icon without text, cleaner icon code)

If there are no changes, feel free to merge this. Two next issues I will work on will be within the LF MU Blocks.
